Generated index.js file does not respect destDir build argument

Steps to reproduce:

1. Generate new project
  
    ```
    $ napi new @test/test
    ```

2. Specify `destDir` argument for `napi build` command:
  
   ```
   $ cd test
   $ napi build --platform build
   ```

3. Run tests
    
   ```
   npm run test
   ``` 

Result:

```
$ npm run test

> @test/test@0.0.0 test
> ava


  Uncaught exception in __test__/index.spec.mjs

  Error: Cannot find module '@test/test-linux-x64-gnu'
  Require stack:
  - /home/kostya/code/test/index.js

  › - index.js
  › Object.<anonymous> (index.js:174:31)

  ✖ __test__/index.spec.mjs exited with a non-zero exit code: 1
  ─

  1 uncaught exception
```

--- a/cli/src/build.ts
+++ b/cli/src/build.ts
@@ -600,7 +600,7 @@ export class BuildCommand extends Command {
         this.jsBinding &&
         this.jsBinding !== 'false' &&
         this.appendPlatformToFilename
-          ? join(process.cwd(), this.jsBinding)
+          ? join(process.cwd(), this.destDir ?? '.', this.jsBinding)
           : null
       const idents = await processIntermediateTypeFile(
         intermediateTypeFile,

